install/bootstrap_livecd_usb.md

```diff
@@ -305,6 +305,9 @@ Some files are needed for generating the configuration payload. See these topics
 
    1. Generate the system configuration.
 
+   > **`NOTE`** ensure you select a reachable NTP pool/server passed in via the `--ntp-pools`/`--ntp-servers` flags, respectively.  Adding an uncreachable server can cause clock skew as chrony tries to continually reach out to a server it can never reach.
```

Comment:
Two spaces after period of first sentence.

Reply:
fixed